packages/terra-data-grid/src/subcomponents/Row.jsx

```diff
@@ -141,13 +124,13 @@ function Row(props) {
       columnIndex={0}
       isSelected={isSelected}
       ariaLabel={ariaLabel}
-      onCellSelect={handleCellSelect}
+      onCellSelect={onCellSelect}
     />
   ) : null;
 
   return (
     <tr
-      className={cx('worklist-data-grid-row', {
+      className={cx('data-grid-row', {
```

Comment:
I would just make the class for this element, "row", instead of "data-grid-row".  The Row component will be a subcomponent of Terra Table.  With class mangling, there is no need for a data-grid prefix and will lead to another rename.

Reply:
Updated